entities/schema/platformics.yaml

```diff
@@ -109,11 +109,6 @@ enums:
       level_phylum:
       level_kingdom:
       level_superkingdom:
-  PhylogeneticTreeFormat:
```

Comment:
Removing PhylogeneticTree as discussed in a NextGen meeting. This is because the phyloTree in CG is just a temporary file from the user we upload and send to Nextclade, and we don't show it/do anything else with it once we redirect the user to Nextclade.